tests/.aspell.en.pws

```diff
@@ -12,3 +12,8 @@ Repo
 screenshots
 wtd
 yml
+armhfp
```

Comment:
please rewrite the whole file into alphabetical order
remove "rebuilded" as this word does not exist.
The attributes in the readme file should have multiline code surrounding them, so maybe epel and repo are also not needed here.

>